scripts/cibuildpkg.py

```diff
@@ -303,6 +306,11 @@ def _extract(self, package: Package) -> None:
         if os.path.exists(patch):
             run(["patch", "-d", path, "-i", patch, "-p1"])
 
+        # Config.guess script is old in "libtheora".
```

Comment:
This should not be needed, "cibuildpkg.py" now automatically updates config.guess and config.sub